styleguide/derek/index.jade

```diff
@@ -1,13 +1,36 @@
 extends ../_layouts/page.jade
 
 block page
-    h1 Derek Theme
+  .container.standard-padding
+    .row.half-padding-full
+      .col-md-12
+        h4.red.no-margin Welcome to the
+        h2.no-margin Global Ecomm Theme
+        h4.red.no-margin Derek Zoolander
 
-    p Common components found throughout Rackspace web properties.
+    .row.half-padding-full
+      .col-md-12
+        p If you want all the global elements stacked with Global Ecomm's patterns, modifiers, global elements and all the goodies just copy the links below
+        code.language-javascript
+              = '<link href="https://043a96a8a2caadd9c57463-9701ce6e6cbd7003c5a8a25a5a69546c.ssl.cf5.rackcdn.com/v1.1/global/css/global.css" />'
+              <br/>
```

Comment:
The `<br />` tag should be in jade. My vagueness wasn't specific enough?  Pfft.  :-p